foodsaving/users/templates/accountdelete_request-body-text.jinja

```diff
@@ -1,12 +1,13 @@
-{% trans full_name=user.get_full_name(), url=resolve_front_url('delete-account', user.delete_token) %}
+{% load i18n %}
+{% blocktrans with full_name=user.display_name %}
 Hello {{ full_name }},
-we're sad to hear that you want to delete your account on foodsaving.world.
-To finally and irreversibly delete your account click the following link:
+We're sad to hear that you want to delete your account on foodsaving.world.
+To finally and irreversibly delete your account, click the following link:
 
 {{ url }}
```

Comment:
They are passing the URL via context, using Django's `reverse`. We would write our own `resolve_front_url` instead:

https://github.com/getsentry/sentry/blob/master/src/sentry/models/organizationmember.py#L163